### PR TITLE
Local minimum resolver and "How do I determine the instrumental noise level in my data?" section

### PR DESCRIPTION
**Pull request by titodamiani**

### CODE REVIEW

**Comment by titodamiani** [comment]
The https://code.google.com/p/mztab/wiki/jmzTab link in the wikiacknowledgements.md file is broken, but I haven't added it. Don't know where it comes from

**Comment by titodamiani** [comment]
ERROR    -  Missing 1 link(s):
WARNING  -    | #module_docs/featdet_adap_chromatogram_builder/adap-chromatogram-builder/:min-group-size-in--of-scans

The link is actually correct, I think

**Comment by titodamiani** [comment]
ERROR - Missing 1 link(s):
WARNING - | #module_docs/featdet_adap_chromatogram_builder/adap-chromatogram-builder/:min-group-size-in--of-scans

The link works. It just doesn't like the '#' character in the title of the subsection linked (i.e. Min group size in # of scans)

**Comment by SteffenHeu** [comment]
Maybe we should replace that by "number" then and see if we can fix the issue that way.

**Comment by titodamiani** [comment]
I already tried and it works. So that's the problem. Will change it later today. Will you also change the name of the parameter in MZmine to avoid weird characters? Or it will stay as it is?

**Comment by SteffenHeu** [comment]
I think in MZmine itself we use the "#" to keep the parameter names short and the windows compact. We could change it, but i think everyone should be able to tell that it's the same parameter.

File: docs/module_docs/featdet_resolver_local_minimum/local-minimum-resolver.md [hunk 1]
```diff
@@ -1,109 +1,88 @@
...
 
-During the EICs building, overalpping and partially co-eluting peaks are retained as single features in the faeture list (see, for example, [ADAP chromatogram builder](../featdet_adap_chromatogram_builder/adap-chromatogram-builder.md)). The _local minimum resolver_ module aims at splitting such "shoulder" LC peaks into individual features (_i.e._ [chromatographic deconvolution](../../terminology/general-terminology.md#chromatographic-resolving)) based on local minima. In fact, a local minimum in the EIC trace might correspond to the valley between two adjacent, partially-resolved peaks.
+During the EICs building, overalpping and partially co-eluting peaks are retained as single features in the faeture list (see, for example, [ADAP chromatogram builder](../featdet_adap_chromatogram_builder/adap-chromatogram-builder.md)). The _local minimum resolver_ module aims at splitting such "shoulder" LC peaks into individual features (_i.e._ [chromatographic resolving](../../terminology/general-terminology.md#chromatographic-resolving)) based on local minima. In fact, a local minimum in the EIC trace might correspond to the valley between two adjacent, partially-resolved peaks.
```
Comment (SteffenHeu) [inline docs/module_docs/featdet_resolver_local_minimum/local-minimum-resolver.md:3]: faeture -> feature

File: docs/module_docs/featdet_resolver_local_minimum/local-minimum-resolver.md [hunk 2]
```diff
@@ -1,109 +1,88 @@
...
+The algorithm examines all the data points in the EIC trace starting from the earliest RT. A scan window is set (see [Minimum search range RT/Mobility](#minimum-search-range-rtmobility-absolute) parameter) and centred around the examined data point. A data point is considered a local minimum if it is the lowest intense point within the scan window. When a local minumum is found, a set of user-defined intensity and peak duration requirements is checked. If they are fulfilled, the original overlapping peaks are split into new, distinct features. The LMR is particularly suitable for LC-MS data with little noise and nice peak shapes.
```
Comment (SteffenHeu) [inline docs/module_docs/featdet_resolver_local_minimum/local-minimum-resolver.md:5]: centred  -> centered

File: docs/module_docs/featdet_resolver_local_minimum/local-minimum-resolver.md [hunk 3]
```diff
@@ -1,109 +1,88 @@
...
 
-Other parameters such as minimum intensities or minimum number of data points should be adjusted
-depending on what you want to detect.
+4. Different vendors use different units of mobility. For instance, [TIMS](../../terminology/ion-mobility-terminology.md#trapped-ion-mobility-spectrometry-tims) express ions' mobility as Vs/cm<sup>2</sup>, whereas [time dispersive IM devices](../../terminology/ion-mobility-terminology.md#time-dispersve-ion-mobility-spectrometry-dtims-and-twims) (DTIMS and TWIMS) use the ions' drift time (expressed in milliseconds). TIMS values are numerically smaller (_e.g._ 0.01) than DTIMS or TWIMS (_e.g._ 0.05); therefore, the [minimum search range](#minimum-search-range-rtmobility-absolute) parameter should be adjusted accordingly.
```
Comment (SteffenHeu) [inline docs/module_docs/featdet_resolver_local_minimum/local-minimum-resolver.md:88]: 0.05 -> 1 for twims/dtims
twims dimension is something in the range of 10-30 ms
dtims is something in the range of 10-60 ms

**Review by SteffenHeu — CHANGES_REQUESTED** [review]
only small typos

**Comment by SteffenHeu** [comment]
Cheers!

**Pull request merged**